packages/react-components/react-dialog/src/components/DialogTrigger/useDialogTrigger.ts

```diff
@@ -1,7 +1,8 @@
 import * as React from 'react';
 import { useModalAttributes } from '@fluentui/react-tabster';
-import { applyTriggerPropsToChildren, getTriggerChild, useEventCallback } from '@fluentui/react-utilities';
-import { DialogTriggerChildProps, DialogTriggerProps, DialogTriggerState } from './DialogTrigger.types';
+import { applyTriggerPropsToChildren, getTriggerChild } from '@fluentui/react-utilities';
+import { useEventCallback } from '@fluentui/react-utilities';
```

Comment:
Why were these split into two?

```suggestion
import { applyTriggerPropsToChildren, getTriggerChild, useEventCallback } from '@fluentui/react-utilities';
```

Reply:
IT was a stash from my previous branch where I had the `react-trigger` package 😅, I just updated the name of the import forgot to verify this. Thx for the review!